repository_service_tuf_worker/repository.py

```diff
@@ -2209,8 +2221,8 @@ def metadata_delegation(
                 delegations: Delegations = Delegations.from_dict(
                     payload["delegations"]
                 )
-                targets = self._storage_backend.get(Targets.type)
-                snapshot = self._storage_backend.get(Snapshot.type)
+                targets = self._storage_load_targets()
```

Comment:
I will create a new good-first-issue to refactor the code using these help functions